langchain/callbacks/manager.py

```diff
@@ -81,11 +83,11 @@ def tracing_v2_enabled(
     )
     if isinstance(example_id, str):
         example_id = UUID(example_id)
-    cb = LangChainTracerV2(example_id=example_id)
-    session = cast(TracerSessionV2, cb.new_session(session_name))
-    tracing_callback_var.set(cb)
```

Comment:
Was the original intentional? Seems like it should be the v2 callback var here?

Reply:
yup this was a msitake good catch